src/View3d.ts

```diff
@@ -35,10 +36,21 @@ export interface View3dOptions {
   useWebGL2?: boolean;
 }
 
+const allGlobalLoadingOptions = {
+  numChunksToPrefetchAhead: 5,
```

Comment:
Is this value always 5? Would making it dynamically determined provide more flexibility for different use cases?

Reply:
Yes and we will probably be adding some better user-controls for this in the future.